### DISCUSSION TITLE
hidden class goes crazy

### DISCUSSION THREAD

**Discussion by pedroelinformatico** [tailwindlabs/tailwindcss#4777] [Help] [answered]
hi, I'm observing and I don't know why it happens, but in a responsive menu, putting class = "hidden" which is supposed to be for small screens in google chrome does not work in other browsers yes

I pass a menu copied from tailwindui https://tailwindui.com/components/application-ui/application-shells/stacked that works in the original web but you take the example and at least after many tests I pass the url where it does not work https://hiddenprueba2021.000webhostapp.com/

I repeat it does not work in chrome
someone can help me, thanks

**Comment by olets** [comment, marked as answer]
Do you mean the element that starts `<div class="md:hidden" id="mobile-menu">` is not showing for you on mobile Chrome? It is for me

![image](https://user-images.githubusercontent.com/3282350/124040916-90f6fd80-d9a1-11eb-9d85-5c5e1bbdeced.png)

**Reply by pedroelinformatico** [reply]
Correct, the problem was, in a plugin that I had installed in my chrome browser, thank you very much for your answer
